allennlp/data/vocabulary.py

```diff
@@ -3,13 +3,18 @@
 
 import codecs
 import logging
-import tqdm
+import os
 
 from allennlp.common.util import namespace_match
 
+import tqdm
+
 logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
 
 DEFAULT_NON_PADDED_NAMESPACES = ("*tags", "*labels")
+DEFAULT_PADDING_TOKEN = "@@PADDING@@"
+DEFAULT_OOV_TOKEN = "@@UNKNOWN@@"
```

Comment:
In order to use them as default arguments to methods in Vocabulary, they need to be global, not class variables.  Two of these are used as default arguments.  The other two are there for consistency.